src/controllers/totp.ts

```diff
@@ -79,13 +83,17 @@ export const postTotpConfigurationController = async (
       throw new NotFoundError();
     }
 
+    const temporaryForce2fa = getTemporaryForce2Fa(req);
```

Comment:
question: To me, temporaryForce2fa is always false here as within the dashboard, the "force_2fa" tik is located elsewhere. Am I missing something ?

Reply:
ligne 92 je remplace temporaryForce2fa par true